tools/wtdbg/wtdbg.xml

```diff
@@ -0,0 +1,195 @@
+<tool id="wtdbg" name="wtdbg" version="0.1.0">
```

Comment:
description is missing

Also we need to add to all out tool `version_command`

Reply:
what is it used for?

Reply:
https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-version-command

Reply:
-help + grep?